source/js/main.js

```diff
@@ -379,6 +381,95 @@ let main = {
       );
     });
 
+    function profileCardSocialAnalytics(socialTwitter, socialLinkedIn) {
+      if (socialTwitter) {
+        socialTwitter.addEventListener(`click`, evt => {
+          evt.preventDefault();
```

Comment:
we don't want this line here anymore, because it turns off the link

Reply:
removed it